VATSpy.dat

```diff
@@ -17524,46 +17524,48 @@ ZBYZ|Bayannaoer Tianjitai|40.926944|107.739444||ZBPE|0
 ZBZJ|Zhangjiakou Ningyuan|40.739167|114.930833||ZBPE|0
 ZBZL|Zhalantun Genghis Khan|47.866667|122.768889||ZBPE|0
 ZGBH|Beihai Fucheng|21.539444|109.293889||ZGZJ|0
-ZGBS|Baise Bama|23.719444|106.959167||ZGNN|0
-ZGCD|Changde Taohuayuan|28.918889|111.640278||ZGHA|0
-ZGCJ|Huaihua Zhijiang|27.441389|109.699722||ZGHA|0
-ZGCS|Changsha Datuopu|28.199720|113.083060||ZGHA|0
-ZGDY|Zhangjiajie Hehua|29.102778|110.443333||ZGHA|0
-ZGFS|Foshan Shadi|23.083333|113.070833||ZGGG|0
-ZGGG|Guangzhou Baiyun|23.392500|113.298889||ZGGG|0
-ZGHA|Changsha Huanghua|28.196667|113.220833||ZGHA|0
-ZGHC|Hechi Jinchengjiang|24.805|107.710556||ZGGG|0
-ZGHY|Hengyang Nanywe|26.722222|112.618056||ZGGG|0
-ZGHZ|Huizhou Pingtan|23.048611|114.600278||ZGGG|0
-ZGKL|Guilin Liangjiang|25.216667|110.040000||ZGKL|0
-ZGLD|Luoding|22.710833|111.601389||ZGGG|0
+ZGBS|Baise Bama|23.719444|106.959167||ZGZU|0
+ZGCD|Changde Taohuayuan|28.918889|111.640278||ZGZU|0
+ZGCJ|Huaihua Zhijiang|27.441389|109.699722||ZGZU|0
+ZGCZ|Chenzhou Beihu|25.753611|112.845556||ZGZU|0

```

Comment:
Airports are sorted alphabetically, this entry needs to be moved one down.

Reply:
done, first time to do that. sorry for the trouble